crates/nu-parser/src/parse_keywords.rs

```diff
@@ -2286,6 +2286,28 @@ pub fn parse_let(
                             parse_var_with_opt_type(working_set, &spans[1..(span.0)], &mut idx);
                         error = error.or(err);
 
+                        let var_name =
+                            String::from_utf8_lossy(working_set.get_span_contents(spans[1]))
+                                .to_string();
+
+                        if var_name == "in" {
```

Comment:
What about using the following, I think it can reduce lines
```rust
if ["in", "nu", "env", "nothing"].contains(&var_name.as_str()) {
    error = error.or_else(...)
}
```

Reply:
That's a great suggestion, thanks!